spark/src/main/java/org/opensearch/sql/spark/asyncquery/model/SparkSubmitParameters.java

```diff
@@ -77,6 +78,11 @@ public Builder className(String className) {
       return this;
     }
 
+    public Builder clusterName(String clusterName) {
+      config.put(FLINT_INDEX_STORE_CLUSTER_NAME_KEY, clusterName);
```

Comment:
Can we make this an environment variable instead of constant, so that we don't need to pass this variable at all places.
For if we set below variables.
spark.emr-serverless.driverEnv.FLINT_CLUSTER_NAME
spark.executorEnv.FLINT_CLUSTER_NAME

I can simply use System.getEnv().get(FLINT_CLUSTER_NAME) in both executor and driver to get the clustername.